esp/public/media/scripts/custom_form.js

```diff
@@ -1190,10 +1271,16 @@ var submit=function() {
 	
 	var form={'title':$j('#form_title').html(), 'desc':$j('#form_description').html(), 'anonymous':($j('#id_anonymous').prop('checked') == "checked"), 'pages':[]}, section, elem, page, section_seq, page_seq=0;
 	form['link_type']=$j('#links_id_main').val();
-	if(form['link_type']!='-1' && $j('#links_id_specify').val()=='particular')
+	if(form['link_type']!='-1' && $j('#links_id_specify').val()=='particular'){
 		form['link_id']=$j('#links_id_pick').val();
-	else form['link_id']=-1;	
-	
+        if($j("#links_id_main").val()=="Program"){
```

Comment:
I don't think this was your doing, but when you pick which program in the "links" section, the list of programs has the oldest at the top instead of the newest at the top (which seems to be the convention everywhere else on the site, and in the dropdown for programs in form permissions)